citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -86,9 +87,21 @@ public IImportedAuthority retrieveAuthorityData(String uri) throws URISyntaxExce
         }
         return null;
     }
-    
+
     @Override
     public String getId() {
         return ID;
     }
+
+    @Override
+    public AuthoritySearchResult searchAuthorities(String firstName, String lastName, int page, int pageSize)
+            throws UnsupportedOperationException {
```

Comment:
since this is a runtime exception it doesn't need to be declare (which is the point here, since you don't want to change all your code when it is implemented).